app/views/user_mailer/invitation.text.erb

```diff
@@ -7,11 +7,13 @@
 <%= t ".been_invited" %>
 <% end %>
 
-<%= t ".part1" %>
+<%= t ".text.ao3_intro" %>
 
-<%= t ".part2" %>
+<%= t ".account_functionality" %>
 
-<%= t ".text.part3", invitation_link: signup_url(:invitation_token => @invitation.token) %>
+<%= t ".text.invite_link", invitation_link: signup_url(:invitation_token => @invitation.token) %>
```

Comment:
As long as you're here, could you change `invitation_link` to `invitation_url`, `support_link` to `support_url`, and the second `faq_link` to `faq_url`? 

As our [i18n Standards](https://github.com/otwcode/otwarchive/wiki/Internationalization-(i18n)-Standards) mention, we prefer to use `_url` as the variable name when we're just dealing with URLs and reserve `_link` for hyperlinks.